pkg/runtime/plan/union.go

```diff
@@ -27,11 +27,14 @@ import (
 
 import (
 	"github.com/arana-db/arana/pkg/proto"
+	"github.com/arana-db/arana/pkg/transformer"
 )
 
 // UnionPlan merges multiple query plan.
 type UnionPlan struct {
 	Plans []proto.Plan
+	transformer.Combiner
+	AggrLoader *transformer.AggrLoader
```

Comment:
Create a brand-new 'AggregatePlan', so we can make complex plan over mixing different plans. 
For example:
```go
type AggregatePlan struct {
    UnionPlan
    // transformer, aggr ...
}
```

Reply:
will open a new branch fix it